### PR TITLE
Add docs for validation forms with djmoney

### PR DESCRIPTION
**Pull request by leportella**
It took me a while to figure this out, so I decided to add an example to the docs :)

### CODE REVIEW

**Review by rougeth — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -140,6 +140,32 @@ The ``balance`` field from the model above has the following validation:
 * Euros should be between 100 and 1000;
 * US Dollars should be between 50 and 500;
 
+Validation with forms
+---------------------
+
+When using with the default ModelForm validation, it is expected that you should
+pass amount and currency as two separate fields:
```
Comment (benjaoming) [inline README.rst:147]:
```suggestion
Constructing form data
----------------------

The default ``ModelForm`` class will use a form field (``djmoney.forms.fields.MoneyField``) that is constructed of two separate fields for amount and currency.

If you need to feed data directly to such a form (for instance if you are writing a test case), then you need to pass amount and currency like this:

```

File: README.rst [hunk 2]
```diff
@@ -140,6 +140,32 @@ The ``balance`` field from the model above has the following validation:
...
+                default_currency='EUR'
+            )
+
+        # forms.py
+        class ProductForm(ModelForm):
+            class Meta:
+                model = Product
+                fields = "__all__"
+
+        ProductForm({'price_0': 10, 'price_1': 'EUR'})
```
Comment (benjaoming) [inline README.rst:166]:
```suggestion
        # tests.py

        # construct the form in your test case
        form = ProductForm({'price_0': 10, 'price_1': 'EUR'})
```

File: README.rst [hunk 3]
```diff
@@ -140,6 +140,32 @@ The ``balance`` field from the model above has the following validation:
...
+                max_digits=14,
+                decimal_places=2,
+                default_currency='EUR'
+            )
+
+        # forms.py
+        class ProductForm(ModelForm):
+            class Meta:
+                model = Product
+                fields = "__all__"
```
Comment (benjaoming) [inline README.rst:164]:
```suggestion
                fields = ["price"]
```

**Review by benjaoming — APPROVED** [review]
I would like to make these changes to make it easier to understand what it's written for.... which I'm guessing is something with constructing tests.

**Comment by benjaoming** [comment]
@leportella thanks for addressing the feedback, will check that the docs build works before merging. We've just recently enabled PR builds.

**Pull request merged**